cpp/OcvPersonDetection/plugin-files/descriptor/descriptor.json

```diff
@@ -60,11 +60,23 @@
         },
         {
           "name": "MERGE_TRACKS",
-          "description": "When set to true, the detection response handler will merge tracks spanning segment boundaries.",
+          "description": "In the context of videos, when set to true, the detection response handler will merge tracks spanning segment boundaries.",
```

Comment:
Let's simplify:

"In the context of videos, when set to true, attempt to merge tracks spanning segment boundaries."